packages/components/src/components/modal/modal.scss

```diff
@@ -47,6 +47,24 @@
             flex-grow: 1;
             padding: 16px;
             line-height: 24px;
+            display: flex;
+            align-items: center;
+        }
+        &__main--no-sidebar {
+            @include themify($themes) {
```

Comment:
@reza-fs can you add this in trader where you have sidebar in modal?

Reply:
Yeah, when I started working on this branch, vertical-tab and modal were tangled together. After your changes, it is irrelevant now. I remove it.